spec/hosts/roles/hexrot_spec.rb

```diff
@@ -32,16 +32,85 @@
           it { is_expected.to contain_class('docker') }
 
           %w[
+            anaconda
+            mate
```

Comment:
This duplicates `          it { is_expected.to contain_class('mate') }`

Reply:
removed/changed 👍🏻